src/foremast/s3/s3deploy.py

```diff
@@ -68,8 +68,10 @@ def setup_pathing(self):
         s3_format = "s3://{}"
         version_path = path_format.format(self.bucket, self.s3path, self.version).replace('//', '/')
```

Comment:
Somehow, this continous `.replace('//', '/')` seems strange. Maybe there's a way to do this earlier or replace with a function?